frontend/src/components/filter/FilterMenu.js

```diff
@@ -171,7 +171,7 @@ export default function FilterMenu({
 
   return (
     <DropdownMenu
-      buttonText="Filters"
+      buttonText={`Filters${filters && filters.length > 0 ? ` (${filters.length})` : ''}`}
       buttonAriaLabel="open filters for this page"
```

Comment:
Do we need to add the number of filters to the aria label as well?

Reply:
This has been updated.